routes/notificationRouter.js

```diff
@@ -1,8 +1,83 @@
-// const express = require('express');
-// const { db } = require('../server/db');
+const express = require('express');
+const { db } = require('../server/db');
 
-// const notificationRouter = express.Router();
+const notificationRouter = express.Router();
 
-// Your code below (delete this line when committing):
+notificationRouter.get('/', async (req, res) => {
+  try {
+    const allNotifications = await db.query('SELECT * FROM notification;');
+    res.status(200).send(allNotifications);
+  } catch (err) {
+    res.status(500).json({
+      status: 'Failed',
+      msg: err.message,
+    });
+  }
+});
 
-// module.exports = notificationRouter;
+notificationRouter.get('/:id', async (req, res) => {
+  const { id } = req.params;
+  try {
+    const idNotification = await db.query('SELECT * FROM notification WHERE business_id = $(id);', {
+      id,
+    });
+    res.status(200).send(idNotification);
+  } catch (err) {
+    res.status(500).json({
+      status: 'Failed',
+      msg: err.message,
+    });
+  }
+});
+
+notificationRouter.post('/', async (req, res) => {
+  const { business_id: businessId, message, timestamp, been_dismissed: beenDismissed } = req.body;
+  try {
+    await db.query(
+      `
+        INSERT INTO notification (business_id, message, timestamp, been_dismissed)
+        VALUES
+        ($(businessId), $(message), $(timestamp), $(beenDismissed));
+      `,
+      { businessId, message, timestamp, beenDismissed },
+    );
+    res.status(200).json({
+      status: 'Success',
+    });
+  } catch (err) {
+    res.status(500).json({
+      status: 'Failed',
+      msg: err.message,
+    });
```

Comment:
The way you did it is technically correct, however, it is redundant with indicating that `status: 'Failed'`, do it like so `res.status(500).send(err.message);`.